scripts/create-github-release.js

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-extraneous-dependencies */
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This script creates a release on GitHub for the Backstage repository.
+ * Given a git tag, it identifies the PR created by changesets which is responsible for creating
+ * the git tag. It then uses the PR description consisting of changelogs for packages as the
+ * release description.
+ *
+ * Example:
+ *
+ * (Dry Run mode, will create a DRAFT release, but will not publish it.)
+ * (Draft releases are visible to maintainers and do not notify users.)
+ * $ node scripts/get-release-description v0.4.1 <GITHUB_TOKEN>
+ *
+ * This will open the git tree at this tag https://github.com/backstage/backstage/tree/v0.4.1
+ * It will identify https://github.com/backstage/backstage/pull/3668 as the responsible changeset PR.
+ * And will use everything in the PR description under "Releases" section.
+ *
+ * (Production or GitHub Actions Mode)
+ * $ node scripts/get-release-description v0.4.1 <GITHUB_TOKEN> 1
+ *
+ * This will do the same steps as above, and will publish the Release with the description.
+ */
+
+const { Octokit } = require('@octokit/rest');
+
+// See Examples above to learn about these command line arguments.
+const [TAG_NAME, GITHUB_TOKEN, BOOL_CREATE_RELEASE] = process.argv.slice(2);
```

Comment:
Let's pass the token through env instead of args, in case it's logged somewhere

Reply:
Done @Rugvip ! Over to you now. Try running the script for previous versions. :)